var/spack/repos/builtin/packages/py-rapidfuzz/package.py

```diff
@@ -12,8 +12,14 @@ class PyRapidfuzz(PythonPackage):
     homepage = "https://github.com/maxbachmann/rapidfuzz"
     pypi = "rapidfuzz/rapidfuzz-1.8.2.tar.gz"
 
+    version("3.3.1", sha256="6783b3852f15ed7567688e2e358757a7b4f38683a915ba5edc6c64f1a3f0b450")
     version("1.8.2", sha256="d6efbb2b6b18b3a67d7bdfbcd9bb72732f55736852bbef823bdf210f9e0c6c90")
 
-    depends_on("python@2.7:", type=("build", "link", "run"))
+    depends_on("python", type=("build", "link", "run"))
+    depends_on("py-setuptools@42:", when="@3:", type="build")
     depends_on("py-setuptools", type="build")
-    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-scikit-build@0.17", when="@3:", type="build")
+
+    # CMakeLists.txt
+    depends_on("cmake@3.12:", type="build")
+    depends_on("ninja", type="build")
```

Comment:
- Don't we need a `py-cython` dependency here as well?
- According to [CMakeLists.txt#L1](https://github.com/maxbachmann/RapidFuzz/blob/v3.3.1/CMakeLists.txt#L1) there is also an upper limit to cmake of `3.24`
- `rapidfuzz@3.4.0` is also out, in case you want to update to the newest version.

Reply:
* I don't see cython listed as a dep, I'm guessing it comes pre-cythonized. Could add so we can recythonize though
* I think that's only the [max policy](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html) version allowed, not sure what that means though